tasks/main.yml

```diff
@@ -40,14 +62,46 @@
     - { name: "docker-py", version: "1.10.5" }
     - { name: "docker-compose", version: "1.9.0" }
 
-- name: start docker
-  service: name=docker state=started enabled=yes
+- name: format docker storage
+  filesystem:
+    dev: "{{ docker_storage_device }}"
+    fstype: xfs
```

Comment:
We could explicitly install the dependencies for setting "fstype: xfs". It seems these are not installed at the moment. There's an assumption here that xfs is available out of the box.

Reply:
Yeah, this really assumes it is running on cPouta CentOS image at the moment.